modules/ztp-create-factory-hub-cluster.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="preparing-the-factory-install-environment_{context}"]
+= Preparing the factory install environment
+
+.Prerequisites
+Before you can provision edge clusters at scale, you must meet the following prerequisites:
+
+=== Base
```

Comment:
I think that we leave room for complex prerequisites in procedures, but I don't think that this looks right for listing prerequisites in a concept module.

I think that you should probably just get rid of the .Prerequisites heading. And perhaps consider updating the subheadings to add prereqs there, like "Base prerequisites" and "Networking prerequisites", etc.

Also, do you want these subheadings actually showing up in the TOC? If not, then you can add [discrete] above each one to prevent that. But if you keep as is, then they need to have explicit anchors defined above them. Also, they're skipping a level, going from 1->3, which is showing the following errors during a build:

```
asciidoctor: WARNING: modules/ztp-create-factory-hub-cluster.adoc: line 11: section title out of sequence: expected level 2, got level 3
asciidoctor: WARNING: modules/ztp-create-factory-hub-cluster.adoc: line 18: section title out of sequence: expected level 2, got level 3
asciidoctor: WARNING: modules/ztp-create-factory-hub-cluster.adoc: line 35: section title out of sequence: expected level 2, got level 3
```